packages/public/server/src/module/Entity/src/Entity/Controller/EntityController.php

```diff
@@ -86,7 +86,41 @@ public function unrevisedAction()
             }
         }
 
-        $view = new ViewModel(['revisionsBySubject' => $revisionsBySubject]);
+        return $revisionsBySubject;
+    }
+
+    protected function getReviewHelpLinks()
+    {
+        $instance = $this->getInstanceManager()->getInstanceFromRequest();
+
+        if ($instance->getLanguage()->getCode() == "de") {
+            $reviewHelpUrl = "/140479";
+        } else {
+            $reviewHelpUrl = "https://docs.google.com/document/d/1p03xx2KJrFw8Mui4-xllvSTHcEPi8G1bdC8rGXcH6f8/edit";
```

Comment:
IMHO: should be on `serlo.org` in the long-term (@yolo-bird) since it might seem kinda sketchy if we link to Google Docs. This is fine for internal stuff, official community documents should be on the platform itself.

Reply:
It's planned to do an version on en.serlo.org @yolo-bird , right?

Reply:
> IMHO: should be on `serlo.org` in the long-term (@yolo-bird) since it might seem kinda sketchy if we link to Google Docs. This is fine for internal stuff, official community documents should be on the platform itself.

The google doc will only be active until the end of february. It's is a nice and easy way to get feedback for the most urgent wishes of reviewers. If @inyono  does not feel great about linking to google docs at all, we'll have to think of another solution.

@kulla  we will add english "subtitles" to the german form, so that at the end it will be bilingual. Hence it will be the same link for every language version. The "subtitles" will be added until this friday.

Reply:
Fine by me if it's only temporary